src/js/player.js

```diff
@@ -4290,6 +4290,35 @@ class Player extends Component {
     return !!this.isAudio_;
   }
 
+  /**
+   * Get the current audioPosterMode state or set audioPosterMode to true or false
+   *
+   * @param {boolean} [value]
+   *         The value to set audioPosterMode to.
+   *
+   * @return {boolean}
+   *         True if audioPosterMode is on, false otherwise.
+   */
+  audioPosterMode(value) {
+
+    value = value && Boolean(value);
+
+    if (value === undefined || value === this.options_.audioPosterMode) {
+      return this.options_.audioPosterMode;
+    }
```

Comment:
if `value` is falsey, this will maintain value as that value, for example `"" || Boolean("")` will return `""` rather than `false`.

Might be better to do a type check below. Something like

```suggestion
    value = value && Boolean(value);

    if (typeof value === 'undefined' || typeof value !== "boolean" || value === this.options_.audioPosterMode) {
      return this.options_.audioPosterMode;
    }

```